api/app/Console/Commands/SyncNewRodFields.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Enums\CandidateRemovalReason;
+use App\Enums\PoolCandidateStatus;
+use App\Models\PoolCandidate;
+use Carbon\Carbon;
+use Illuminate\Console\Command;
+
+class SyncNewRodFields extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'app:sync-rod-fields';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Where the new RoD fields should be non-null, insert an appropriate value';
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle()
+    {
+        $dateNow = Carbon::now();
+        $placedStatuses = PoolCandidateStatus::placedGroup();
+        $removedStatuses = PoolCandidateStatus::removedGroup();
+        $finalDecisionStatuses = [
+            PoolCandidateStatus::QUALIFIED_AVAILABLE->name,
+            PoolCandidateStatus::QUALIFIED_UNAVAILABLE->name,
+            PoolCandidateStatus::QUALIFIED_WITHDREW->name,
+            PoolCandidateStatus::PLACED_CASUAL->name,
+            PoolCandidateStatus::PLACED_TERM->name,
+            PoolCandidateStatus::PLACED_INDETERMINATE->name,
+            PoolCandidateStatus::EXPIRED->name,
+            PoolCandidateStatus::SCREENED_OUT_APPLICATION->name,
+            PoolCandidateStatus::SCREENED_OUT_ASSESSMENT->name,
+        ];
+
+        $applicableModels = PoolCandidate::with('user')
+            ->whereNotIn('pool_candidate_status', [
+                PoolCandidateStatus::DRAFT->name,
+                PoolCandidateStatus::DRAFT_EXPIRED->name,
+                PoolCandidateStatus::NEW_APPLICATION->name,
+                PoolCandidateStatus::APPLICATION_REVIEW->name,
+                PoolCandidateStatus::SCREENED_IN->name,
+                PoolCandidateStatus::UNDER_ASSESSMENT->name,
+            ])
```

Comment:
None of these statuses should be involved, so not fetched in the first place to reduce number of models grabbed and iterated through